### ISSUE TITLE
Hope can add clipboard support

### ISSUE THREAD

**Issue by Bob-YsPan** [minecraft-linux/mcpelauncher-manifest#49]
This project is very useful. I can play old version Minecraft PE and use Windows 10 version control!
but sometimes I may copy some command (like command block) or paste some text in-game (like Chinese characters) but it not works...
Hope can add this function in it...
(Sorry for my bad English... I speak Chinese...)

**Comment by Bob-YsPan** [comment]
I try it on Archlinux.
Desktop environment I use Mate and Gnome 3.
This package I install it from AUR.

When I press "Ctrl + A"(Select all), it appears symbol like "Ã"
"Ctrl + C" and "Ctrl + V" nothing happened(I copied some English text before).
![2019-01-26 18-34-19](https://user-images.githubusercontent.com/46966555/51785988-34240700-2199-11e9-8b9d-dce62c7b10a4.png)
(This image I captured on Mate Desktop)
